crawler/settings.py

```diff
@@ -35,7 +35,11 @@
 }
 
 EXTENSIONS = {
+    'crawler.extensions.sentry.SentryLogging': -1, # Load SentryLogging extension before others
 }
+# Send exceptions to Sentry
+# replace SENTRY_DSN by you own DSN
+SENTRY_DSN = "https://f59068a56b0c420daf5e1680d0276c0b@sentry.io/1817994"
```

Comment:
Please dont't commit any environment setting into git. A typical solution would be read this from environment variable, or from an optional config file.  XD

In addition, it would be better if we can make Sentry to be optional, so even people without knowledge of Sentry, or doesn't want to send info to Sentry, can use this tool.

Reply:
OK I will fix this issue. Thanks for your review

Reply:
Forgot to mention, could you also help to add related description into `README.md`?  Than other people will know now this tool support Sentry! Thanks~

Reply:
OK!!